intrinsicsolver/fixFunctions.go

```diff
@@ -0,0 +1,88 @@
+package intrinsicsolver
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+)
+
+// FixFunctions : takes []byte file and firstly converts all single quotation marks to double ones (anything between single ones is treated as the rune in GoLang), then deconstructs file into lines, checks for intrinsic functions of a map nature where the function name is located in one line and it's body (map elements) are located in the following lines (if this would be not fixed an error would be thrown: `json: unsupported type: map[interface {}]interface {}`). The function changes the notation by putting function name in the next line with proper indentation and saves the result to temporary file, then opens it and returns []byte array.
+func FixFunctions(template []byte) []byte {
```

Comment:
You should also return `error` from this function to be able to handle it outside.